### DISCUSSION TITLE
Expanded row on Table with variant="striped"

### DISCUSSION THREAD

**Discussion by bruno-fialho** [chakra-ui/chakra-ui#4401] [Q&A]
Hello,

I have a react-table with expanded rows, and when I display the expanded row, all the rows until the end of the table switch colors. 

Is there a way to disable striped style for only one <Tr>?

**Comment by primos63** [comment]
"Is there a way to disable striped style for only one ?" I'm not sure I understand what this means.
 
 Have a look at my [CodeSandbox](https://codesandbox.io/s/react-table-with-chakra-ui-updlt). I made another variant based on the striped variant. Maybe this is what you are looking for?
 
 I don't know how this will work with filtering, grouping or sorting. This is my first time playing with React-Table.
